config/src/main/java/com/typesafe/config/impl/ConfigImplUtil.java

```diff
@@ -32,6 +32,14 @@ else if (a == b) // catches null == null plus optimizes identity case
             return a.equals(b);
     }
 
+    public static boolean isC0Control(final char ch) {
```

Comment:
if these are only used within the "impl" package they don't need to be public I guess